### DISCUSSION TITLE
CNAME record not working

### DISCUSSION THREAD

**Discussion by shaagss** [browserpad/browserpad#54] [Q&A] [answered]
I put in "browserpad.org" as a CNAME record for my domain, but it just gave me a Github 404 error
![image](https://user-images.githubusercontent.com/98065104/167129224-5fd445b2-588d-4ce2-9b48-2f9f4f23725b.png)
The website I added it to is http://pad.sha.gs if you want to check

**Comment by waldyrious** [comment, marked as answer]
Hi there. I believe you need to put "pad.sha.gs" in your `CNAME` file. See here for more information: https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site (probably you'll want to follow the "Configuring a subdomain" section).

**Reply by shaagss** [reply]
wouldn't that just make "pad.sha.gs" my github profile?

**Reply by waldyrious** [reply]
I think GitHub Pages is smart enough to set things up so browserpad will live in https://pad.sha.gs/browserpad, whereas the root page at https://pad.sha.gs will point to https://shags6669.github.io. But I can't guarantee that will be the case, as I'm not an expert in GitHub Pages. Please check the GitHub Pages documentation links that I mentioned above for additional assistance.
